Application/EdFi.Ods.AdminApi/Features/ClaimSets/ClaimSetModel.cs

```diff
@@ -46,6 +48,15 @@ public ResourceClaimModel()
     }
 }
 
+[SwaggerSchema(Title = "ResourceClaimActionModel")]
+public class ResourceClaimActionModel
+{
+    public bool Create { get; set; }
+    public bool Read { get; set; }
+    public bool Update { get; set; }
+    public bool Delete { get; set; }
+}
+
 [SwaggerSchema(Title = "SimpleResourceClaimModel")]
 public class SimpleResourceClaimModel
 {
```

Comment:
@jagudelo-gap 
I see so many Resource claim models. Can we refactor to have base and derived classes as needed ?

Reply:
Done. Class names have been changed